tests/acceptance/test_organization_security_privacy.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import
+
+from sentry.testutils import AcceptanceTestCase
+
+
+class OrganizationSecurityAndPrivacyTest(AcceptanceTestCase):
+    def load_organization_helper(self, snapshot_name=None):
+        self.browser.wait_until_not(".loading-indicator")
+        if snapshot_name is not None:
+            self.browser.snapshot("organization settings security and privacy -- " + snapshot_name)
+        assert self.browser.element_exists(".organization-settings-security-and-privacy")
+
+    def renders_2fa_setting(self):
+        return self.browser.element_exists("#require2FA")
+
+    def test_renders_2fa_setting_for_owner(self):
+        with self.feature("organizations:datascrubbers-v2"):
+            user_owner = self.create_user("owner@example.com")
+            organization = self.create_organization(name="Example", owner=user_owner)
+            self.login_as(user_owner)
+            path = "/settings/{}/security-and-privacy/".format(organization.slug)
```

Comment:
Should you also retain tests for the old view as most users will have that for quite a while.

Reply:
Hm, I wasn't sure, but honestly, it's a little difficult for me at the moment to write tests in python, because I don't know much about it

Reply:
Keeping the old file around with no datascubbers feature toggles would likely be enough to keep regression tests around.